Path to 'SetNetworkSharePolicy.ps1' Not Found

**Describe the bug**
When enabling the `storage smb` addon and not having the correct network policies set, the script `SetNetworkSharePolicy.ps1` will be executed. Currently, the path to this script is not found.

**To Reproduce**
Steps to reproduce the behavior:
1. Using a machine not having the appropriate networking polices set
2. Running `k2s addons enable storage smb -o`
3. Read the output
```
Write-ErrorMessage : [11:54:59][ERROR] & : The term 'C:\ws\k2s\addons\storage\smb\module\SetNetworkSharePolicy.ps1' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the
⏳ path is correct and try again.
```

**Expected behavior**
The script `SetNetworkSharePolicy.ps1` is executed without any errors.

--- a/.vscode/spellright.dict
+++ b/.vscode/spellright.dict
@@ -25,3 +25,6 @@ Golang
 mkdocs
 authN
 authZ
+Healthineers
+Goda
+Graphviz

--- a/addons/storage/smb/module/Smb-share.module.psm1
+++ b/addons/storage/smb/module/Smb-share.module.psm1
@@ -41,6 +41,7 @@ $storageClassTimeoutSeconds = 600
 $namespace = 'storage-smb'
 
 $configFilePath = "$PSScriptRoot\..\config\SmbStorage.json"
+$fixNetworkPoliciesScript = "$PSScriptRoot\..\SetNetworkSharePolicy.ps1"
 
 function Test-CsiPodsCondition {
     param (
@@ -240,7 +241,7 @@ function Test-ForKnownSmbProblems {
         gpresult /r /scope:computer | Select-String -Pattern 'OU='
         Write-Log "You can also find the setting here: 'UCMS-ControlCenter' (also called 'User-Client Info') -> 'More details...' -> 'OU' section"
         Write-Log "Please contact your administrator and let them move you to the 'R&D OU' `n"
-        Write-Log "Current workaround: fix the policy issue with powershell <installation folder>\smallsetup\helpers\SetNetworkSharePolicy.ps1 `n"
+        Write-Log "Current workaround: fix the policy issue with PowerShell script '$fixNetworkPoliciesScript' `n"
 
         $script:HasIssues = $true
         return
@@ -312,7 +313,7 @@ function New-SharedFolderMountOnLinuxClient {
 
             if ( $script:HasIssues -eq $true ) {
                 Write-Log 'Executing script to fix policy issue...'
-                & "$PSScriptRoot\SetNetworkSharePolicy.ps1"
+                & $fixNetworkPoliciesScript
             }
         }
         if ($i -ge 6) {
@@ -411,7 +412,7 @@ function Wait-ForSharedFolderMountOnLinuxClient () {
 
             if ( $script:HasIssues -eq $true ) {
                 Write-Log 'Executing script to fix policy issue...'
-                & "$PSScriptRoot\SetNetworkSharePolicy.ps1"
+                & $fixNetworkPoliciesScript
             }
         }
         if ($iteration -ge 20) {

--- a/k2s/internal/README.md
+++ b/k2s/internal/README.md
@@ -1,5 +1,5 @@
 <!--
-SPDX-FileCopyrightText: © 2024 Siemens Healthineers AG
+SPDX-FileCopyrightText: © 2025 Siemens Healthineers AG
 SPDX-License-Identifier: MIT
 -->
 
@@ -16,6 +16,7 @@ The dependencies can be analyzed with *Go* tooling, e.g.:
     ```sh
     go install github.com/loov/goda@latest
     ```
+- Install [*Graphviz*](https://graphviz.org/download/#windows)
 - Generate graph:
     ```sh
     goda graph github.com/siemens-healthineers/k2s/internal/... | dot -Tsvg -o graph-internal.svg

